assets/src/edit-story/components/inspector/stories/checklistTab.js

```diff
@@ -58,8 +58,8 @@ export const _default = () => {
         'Keep the title clear and clean, ideally under 10 words in less than 40 characters.',
     },
     {
-      message: 'Missing Story Description',
-      help: 'Add a Story Description in the Document panel.',
+      message: 'Missing Excerpt',
+      help: 'Add an Excerpt in the Document panel.',
```

Comment:
I vaguely remember updating this to be "story description" because product thought that it was clearer but i can't find the ticket anymore.

Reply:
Hm, took this from Figma -- will double check with UX.

Reply:
Yep, apparently "Story description" should still remain -- reverted that, thanks for noting!